src/Skeleton/Telepes.java

```diff
@@ -46,6 +46,18 @@ public void SetTeleportkapuRakter(Teleportkapu t) {
     	SkeletonController.FunctionReturn();
     }
 
+    /**
+     * az aszteroidan valo furast hajtja vegre
+     */
+    public void Fur() {
+        SkeletonController.FunctionCall(new Object() {
```

Comment:
Ez mi a faszt keres itt?

Reply:
a hajo már nem tud fúrni, mert az ufo nem képes fúrasra, ezért már a legutóbbi osztálydiagramon is úgy van, hogy a fúr a robot és telepes osztályokban van